modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, you should:
+
+* Have access to the cluster as a user with `cluster-admin` privileges.
+* Have a recent etcd backup in case your update fails and you must restore your cluster to a previous state.
```

Comment:
I would double check with an SME, but I think this prereq might be moot if there's nothing the user could do to trigger a new backup. If that's true, this is more like "hope that your most recent backup is recent enough to revert to if disaster recovery is necessary."

One idea might be to keep this here but reword it to something like:
"Check to see that your etcd backup is recent enough in case your update fails and you must restore your cluster to a previous state." Plus maybe a sentence about how etcd periodically creates a new backup on its own?

Reply:
Updated to your suggestion, but wondering if we should specify what "recent enough" means. Leaving this comment thread open to discuss with SMEs.